<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CKTextEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>组:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cob_group">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_group">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="cob_sort">
        <item>
         <property name="text">
          <string>顺序</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>逆序</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>置顶未翻译</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>从短到长</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>从长到短</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="TextTableWidget" name="table">
       <property name="sizeIncrement">
        <size>
         <width>2</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="PropertyWidget" name="tree">
       <property name="baseSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_fit_lineheight">
        <property name="text">
         <string>根据内容刷新行高</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_def_lineheight">
        <property name="text">
         <string>默认行高</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_attr">
        <property name="text">
         <string>编辑属性</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <widget class="QMenu" name="menu_import">
     <property name="title">
      <string>导入</string>
     </property>
     <addaction name="act_json_in"/>
    </widget>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>导出</string>
     </property>
     <addaction name="act_charset"/>
     <addaction name="act_json_out"/>
    </widget>
    <addaction name="act_new"/>
    <addaction name="act_open"/>
    <addaction name="separator"/>
    <addaction name="menu_import"/>
    <addaction name="menu_4"/>
    <addaction name="separator"/>
    <addaction name="act_save"/>
    <addaction name="act_save_as"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助(&amp;A)</string>
    </property>
    <widget class="QMenu" name="menu_language">
     <property name="title">
      <string>语言</string>
     </property>
     <addaction name="act_chinese"/>
    </widget>
    <addaction name="menu_language"/>
    <addaction name="act_about"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="act_popup_edit"/>
    <addaction name="act_auto"/>
    <addaction name="act_transform"/>
    <addaction name="act_show_space"/>
    <addaction name="act_show_line_feed"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_new">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="act_open">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="act_save">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="act_save_as">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="act_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="act_popup_edit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>弹框编辑</string>
   </property>
   <property name="toolTip">
    <string>弹框编辑</string>
   </property>
  </action>
  <action name="act_json_in">
   <property name="text">
    <string>JSON</string>
   </property>
   <property name="toolTip">
    <string>JSON</string>
   </property>
  </action>
  <action name="act_charset">
   <property name="text">
    <string>所有用到的字符</string>
   </property>
   <property name="toolTip">
    <string>所有用到的字符</string>
   </property>
  </action>
  <action name="act_auto">
   <property name="text">
    <string>自动翻译</string>
   </property>
   <property name="toolTip">
    <string>自动翻译</string>
   </property>
  </action>
  <action name="act_json_out">
   <property name="text">
    <string>JSON</string>
   </property>
  </action>
  <action name="act_transform">
   <property name="text">
    <string>转移翻译</string>
   </property>
   <property name="toolTip">
    <string>从其他ckt转移翻译</string>
   </property>
  </action>
  <action name="act_show_space">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示空格</string>
   </property>
   <property name="toolTip">
    <string>显示空格</string>
   </property>
  </action>
  <action name="act_show_line_feed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示换行符</string>
   </property>
   <property name="toolTip">
    <string>显示换行符</string>
   </property>
  </action>
  <action name="act_chinese">
   <property name="text">
    <string notr="true">简体中文</string>
   </property>
   <property name="iconText">
    <string notr="true">简体中文</string>
   </property>
   <property name="toolTip">
    <string notr="true">简体中文</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PropertyWidget</class>
   <extends>QTreeView</extends>
   <header>control/propertywidget.h</header>
  </customwidget>
  <customwidget>
   <class>TextTableWidget</class>
   <extends>QTableView</extends>
   <header>control/text_tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
